server.go

```diff
@@ -67,6 +81,59 @@ func NoDirListing(h http.Handler) http.HandlerFunc {
 	})
 }
 
+func initBlogs() {
+	ArrayOfBlogs = make(map[int]BlogItem)
+	ArrayOfBlogs[0] = BlogItem{Text: "A green landscape", Image: "/img/landscape5.jpg", Timestamp: time.Now().Format("20060102150405"), Time: time.Now().Format("15:04:05")}
+	ArrayOfBlogs[1] = BlogItem{Text: "A mountain landscape", Image: "/img/landscape6.jpg", Timestamp: time.Now().Format("20060102150405"), Time: time.Now().Format("15:04:05")}
+	ArrayOfBlogs[2] = BlogItem{Text: "A lake landscape", Image: "/img/landscape1.jpg", Timestamp: time.Now().Format("20060102150405"), Time: time.Now().Format("15:04:05")}
+	ArrayOfBlogs[3] = BlogItem{Text: "A trees landscape", Image: "/img/landscape2.jpg", Timestamp: time.Now().Format("20060102150405"), Time: time.Now().Format("15:04:05")}
+	ArrayOfBlogs[4] = BlogItem{Text: "A village landscape", Image: "/img/landscape3.jpg", Timestamp: time.Now().Format("20060102150405"), Time: time.Now().Format("15:04:05")}
+	ArrayOfBlogs[5] = BlogItem{Text: "A canyon", Image: "/img/landscape4.jpg", Timestamp: time.Now().Format("20060102150405"), Time: time.Now().Format("15:04:05")}
+	ArrayOfBlogs[6] = BlogItem{Text: "A dog", Image: "/img/Shetland_Sheepdog.jpg", Timestamp: time.Now().Format("20060102150405"), Time: time.Now().Format("15:04:05")}
+	ArrayOfBlogs[7] = BlogItem{Text: "A phone", Image: "/img/silver_nexus.png", Timestamp: time.Now().Format("20060102150405"), Time: time.Now().Format("15:04:05")}
+	ArrayOfBlogs[8] = BlogItem{Text: "Some Sushi", Image: "/img/image3.jpg", Timestamp: time.Now().Format("20060102150405"), Time: time.Now().Format("15:04:05")}
+	ArrayOfBlogs[9] = BlogItem{Text: "Hi Gopher!", Image: "/img/gopher.png", Timestamp: time.Now().Format("20060102150405"), Time: time.Now().Format("15:04:05")}
+}
+
+//returns a map of blogs which contains a number of items specified by the parameter
+func returnMapOfBlogOfSize(size int) map[int]BlogItem {
```

Comment:
The main reason of having this method was that I wanted to update the timestamp. Just doing a slice of the existing array, will return items with the time of their creation. The reason why I need new timestamps, is that, for the second example of amp-live-list I am demonstrating the update capability. This capabiity depends on "data-update-time" parameter: if the server page generates recent values for this parameter, then the component will change.